hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/UIMessageIdProvider.java

```diff
@@ -15,6 +15,8 @@
  */
 public final class UIMessageIdProvider {
 
+    public static final String ARTIFACT_DOWNLOAD = "artifact.download";
```

Comment:
Please add a respective message to the `messages.properties` file (as you did previously: `message.artifact.download.failure = Artifact Download Failed` )